knative-operator/pkg/common/metrics.go

```diff
@@ -15,8 +15,7 @@ var (
 	)
 	KnativeServingUpG  = knativeUp.WithLabelValues("serving_status")
 	KnativeEventingUpG = knativeUp.WithLabelValues("eventing_status")
-	//TODO: to be enabled
-	//KnativeKafkaUpG    = knativeUp.WithLabelValues("kafka_status")
+	KnativeKafkaUpG    = knativeUp.WithLabelValues("kafka_status")
```

Comment:
@aliok if eventing is up does that mean kafka is up?

Reply:
@skonto No, they're separate.